packages/commonwealth/client/scripts/views/components/SublayoutHeader/DesktopHeader/DesktopHeader.tsx

```diff
@@ -111,20 +114,43 @@ const DesktopHeader = ({
 
         {isLoggedIn && (
           <UserDropdown
-            onAuthModalOpen={onAuthModalOpen}
+            onAuthModalOpen={() => onAuthModalOpen}
             onRevalidationModalData={onRevalidationModalData}
           />
         )}
 
         {!isLoggedIn && (
-          <CWButton
-            buttonType="primary"
-            buttonHeight="sm"
-            label="Sign in"
-            buttonWidth="wide"
-            disabled={location.pathname.includes('/finishsociallogin')}
-            onClick={onAuthModalOpen}
-          />
+          <>
+            {userOnboardingEnabled ? (
+              <>
+                <CWButton
+                  buttonType="secondary"
+                  buttonHeight="sm"
+                  label="Create account"
+                  buttonWidth="narrow"
+                  disabled={location.pathname.includes('/finishsociallogin')}
+                  onClick={() => onAuthModalOpen('create-account')}
+                />
+                <CWButton
+                  buttonType="primary"
+                  buttonHeight="sm"
+                  label="Sign in"
+                  buttonWidth="narrow"
+                  disabled={location.pathname.includes('/finishsociallogin')}
+                  onClick={() => onAuthModalOpen()}
+                />
```

Comment:
those buttons are pretty much the same as those in sublayout, so we could extract them to separate component and reuse them